pkg/cmd/grafana-cli/commands/scuemata_validation_command_test.go

```diff
@@ -52,10 +52,51 @@ func TestValidateScuemataBasics(t *testing.T) {
 			DistPluginCueFS: pluginSchema,
 		}
 
-		err := validate(baseLoadPaths, load.BaseDashboardFamily)
+		err := validateScuemata(baseLoadPaths, load.BaseDashboardFamily)
 		require.NoError(t, err, "error while loading base dashboard scuemata")
 
-		err = validate(baseLoadPaths, load.DistDashboardFamily)
+		err = validateScuemata(baseLoadPaths, load.DistDashboardFamily)
 		assert.EqualError(t, err, "all schema should be valid with respect to basic CUE rules, Family.lineages.0.0: field #Panel not allowed")
 	})
+
+	t.Run("Testing scuemata validity against JSON files", func(t *testing.T) {
```

Comment:
We're not trying to test scuemata validity here (though that does happen incidentally); we're trying to test the `validateResources` function against known scuemata and resource inputs.